packages/@react-spectrum/button/stories/Button.stories.tsx

```diff
@@ -83,11 +83,13 @@ storiesOf('Button', module)
   )
   .add(
     'element: a, href: \'//example.com\', target: \'_self\'',
-    () => render({elementType: 'a', href: '//example.com', target: '_self', variant: 'primary'})
+    () => render({elementType: 'a', href: '//example.com', target: '_self', variant: 'primary'}),
+    {note: 'Known browser behavior where keyboard focus in Safari doesn\'t work.'}
```

Comment:
What do you mean "doesn't work" here?

Reply:
I changed the message. The button doesn't get keyboard focus at all.

Reply:
Only if you don't have the setting enabled though right?